quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus.test/projects/maven/config-hover/src/main/java/org/acme/config/GreetingResource.java

```diff
@@ -26,6 +26,9 @@
 
     @ConfigProperty(name = "greeting.number", defaultValue="0")
     int number;
+    
+    @ConfigProperty(name = "${greeting.hover}")
```

Comment:
I would add `greeting.hover` to the application properties used, to confirm that `{greeting.hover}` no longer generates a hover for the existing property.

After that, if there's nothing else, I would merge this.

Reply:
Would this work after saveFile, since a separate test tests the hover?

Reply:
Okay can confirm it works

Reply:
https://github.com/redhat-developer/quarkus-ls/pull/431/files#diff-171d106c3784abda06d0ecbedb2818ded41f5e7223971b5fbdc975884cb1a378R168 should have the same effect, so it ensures I'm using an application.properties file that hasn't been overwritten